cluster-autoscaler/cloudprovider/gce/gce_cloud_provider.go

```diff
@@ -159,6 +159,13 @@ func (gce *GceCloudProvider) NewNodeGroup(machineType string, labels map[string]
 
 // GetResourceLimiter returns struct containing limits (max, min) for resources (cores, memory etc.).
 func (gce *GceCloudProvider) GetResourceLimiter() (*cloudprovider.ResourceLimiter, error) {
+	resourceLimiter, err := gce.gceManager.GetResourceLimiter()
+	if err != nil {
+		return nil, err
+	}
+	if resourceLimiter != nil {
+		return resourceLimiter, nil
+	}
```

Comment:
Why do you keep a resourceLimiter in both GceCloudProvider and GceManager? Wouldn't it be easier to just push the one you get from flage to GceManager instead of keeping it in GceCloudProvider?

Reply:
I store limits from flags in GceCloudProvider.ResourceLimiter. I have changed name.